MSAL/src/native_auth/network/errors/sign_up/MSALNativeAuthRequiredAttributesOptions.swift

```diff
@@ -24,6 +24,6 @@
 
 import Foundation
 
-class MSALNativeAuthRequiredAttributeOptions: Decodable {
+class MSALNativeAuthRequiredAttributesOptions: Decodable {
```

Comment:
This change shouldn't be in the PR now. Also the PR Description still refers to the Attributes change.

Reply:
I've only now removed it.